test/e2e/specs/wp-calypso-gutenberg-editor-tracking-spec.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import assert from 'assert';
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import LoginFlow from '../lib/flows/login-flow.js';
+
+import GutenbergEditorComponent from '../lib/gutenberg/gutenberg-editor-component';
+import WPAdminSidebar from '../lib/pages/wp-admin/wp-admin-sidebar';
+
+import * as driverManager from '../lib/driver-manager.js';
+import * as dataHelper from '../lib/data-helper.js';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+
+// We need to trigger test runs against Gutenberg Edge (the "next" version of Gutenberg that
+// will be deployed to Dotcom) as well as the current version of Gutenberg.
+const gutenbergUser =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
+
+let driver;
+
+before( async function () {
+	this.timeout( startBrowserTimeoutMS );
+	driver = await driverManager.startBrowser();
+} );
+
+function getEventsFiredForBlock( eventsStack, event, block ) {
+	if ( ! eventsStack || ! event || ! block ) {
+		return false;
+	}
+
+	return eventsStack.filter(
+		( [ eventName, eventData ] ) => event === eventName && eventData.block_name === block
+	);
+}
+
+function getTotalEventsFiredForBlock( eventsStack, event, block ) {
+	return getEventsFiredForBlock( eventsStack, event, block ).length;
+}
+
+describe( `[${ host }] Calypso Gutenberg Tracking: (${ screenSize })`, function () {
+	this.timeout( mochaTimeOut );
```

Comment:
Why is `this.timeout( mochaTimeOut );` necessary?

Reply:
I actually don't know but if you remove it everything breaks. It's the same for all the Gutenberg editor tracking files.

@bsessions85 can you shed light on this at all?